api/client/ffs.go

```diff
@@ -0,0 +1,244 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	cid "github.com/ipfs/go-cid"
+	ff "github.com/textileio/powergate/ffs"
+	"github.com/textileio/powergate/ffs/api"
+	pb "github.com/textileio/powergate/ffs/pb"
+)
+
+type ffs struct {
+	client pb.APIClient
+}
+
+// JobEvent represents an event for Watching a job
+type JobEvent struct {
+	Job ff.Job
+	Err error
+}
+
+func (f *ffs) Create(ctx context.Context) (string, string, error) {
+	r, err := f.client.Create(ctx, &pb.CreateRequest{})
+	if err != nil {
+		return "", "", err
+	}
+	return r.ID, r.Token, nil
+}
+
+func (f *ffs) ID(ctx context.Context) (ff.ApiID, error) {
+	resp, err := f.client.ID(ctx, &pb.IDRequest{})
+	if err != nil {
+		return ff.EmptyInstanceID, err
+	}
+	return ff.ApiID(resp.ID), nil
+}
+
+func (f *ffs) WalletAddr(ctx context.Context) (string, error) {
+	resp, err := f.client.WalletAddr(ctx, &pb.WalletAddrRequest{})
+	if err != nil {
+		return "", err
+	}
+	return resp.Addr, nil
+}
```

Comment:
`ID` and `WalletAddr` added for completeness, some overlap with output from `Info`